Rakefile

```diff
@@ -1,34 +1,26 @@
-require 'rubygems'
-require 'rake'
-require 'rake/clean'
 require "bundler/gem_tasks"
+require "rake/clean"
+require "rake/testtask"
+require "rdoc/task"
 
-
-require 'rake/testtask'
-Rake::TestTask.new(:test) do |test|
-  test.libs << 'lib' << 'test'
-  test.pattern = 'test/**/*_test.rb'
-  test.verbose = true
-  test.warning = true
-end
-
-begin
-  require 'rcov/rcovtask'
-  Rcov::RcovTask.new do |test|
-    test.libs << 'test'
-    test.pattern = 'test/**/*_test.rb'
-    test.verbose = true
-  end
-rescue LoadError
-  task :rcov do
-    abort "RCov is not available. In order to run rcov, you must: sudo gem install spicycode-rcov"
+desc "Look for TODO and FIXME tags in the code"
```

Comment:
I'd probably drop this, easy with project-wide search.

But I'm not against keeping it if you prefer.